packages/react/src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -5,125 +5,124 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import PropTypes from 'prop-types';
-import React from 'react';
-import classNames from 'classnames';
 import { settings } from 'carbon-components';
+import cx from 'classnames';
```

Comment:
why abbreviate to `cx`? i typically prefer more readable/explicit names

Reply:
`cx` comes from the "class set" convention from earlier versions of React 👍 Can definitely map it over now if that convention is outdated

Reply:
in this case i usually see it imported as `classnames` around the codebase

Reply:
@lovemecomputer it's definitely a mix across the codebase depending on the author, we can totally consolidate though. `cx` to me is helpful because it's shorter and has been a convention since early versions of React but I understand if that is no longer a relevant reference.